solana/tsconfig.anchor.json

```diff
@@ -3,11 +3,12 @@
   "include": ["ts/sdk/**/*.ts", "ts/sdk/**/*.json"],
   "exclude": ["jest.config.js"],
   "compilerOptions": {
-    "outDir": "dist/esm",
+    "noEmit": true,
```

Comment:
why?

Reply:
its a test tsconfig, no outdir needed and should never emit